### ISSUE TITLE
Trakt redirect URI for personal development key

### ISSUE THREAD

**Issue by SenorSmartyPants** [UweTrottmann/SeriesGuide#639]
I've created my own trakt app to get personal SG_TRAKT_CLIENT_ID and SG_TRAKT_CLIENT_SECRET but I put the wrong information in for the redirect URI. So I get an error because it doesn't match to what seriesguide wants.

What should I enter into my trakt setup for the redirect URI?

**Comment by UweTrottmann** [comment]
Hi there, thanks for your question.

But first, a few words: assume reading your emails in the morning and finding a ton of them from a stranger in your inbox. I bet your first impression of that person is, well, not great. If you then go on to read them only to find out that most of them are simple questions that could have been answered by taking a few minutes to have a look around your project, well, ...

TL;DR: respect that other people's time is as valuable as yours. If you are new to a project, take some time to get to know things.

OK, anyhow happy to help: you can find the callback URI in `BaseOAuthActivity`.
https://github.com/UweTrottmann/SeriesGuide/blob/v47/app/src/main/java/com/battlelancer/seriesguide/traktapi/BaseOAuthActivity.java#L31-L32

**Comment by SenorSmartyPants** [comment]
I'm sorry for my blast of ignorance into your email.

I do appreciate the time you've put into this app. I use it a lot.

I'm successfully signing into trakt now.
